test/TTConfigurationData.py

```diff
@@ -0,0 +1,70 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+from cp3_llbb.Framework import Framework
+
+process = Framework.create(True, eras.Run2_50ns, '74X_dataRun2_v2', cms.PSet(
+    bTagsLoose = cms.PSet(
+        type = cms.string('btags_analyzer'),
+        prefix = cms.string('btags_CSVv2_loose'),
+        enable = cms.bool(True),
+        parameters = cms.PSet(
+            discr_name = cms.untracked.string('pfCombinedInclusiveSecondaryVertexV2BJetTags'),
+            discr_cut = cms.untracked.double(0.605),
+            eta_cut = cms.untracked.double(2.4),
+            pt_cut = cms.untracked.double(30)
+            )
+        ),
+
+    bTagsMedium = cms.PSet(
+        type = cms.string('btags_analyzer'),
+        prefix = cms.string('btags_CSVv2_medium'),
+        enable = cms.bool(True),
+        parameters = cms.PSet(
+            discr_name = cms.untracked.string('pfCombinedInclusiveSecondaryVertexV2BJetTags'),
+            discr_cut = cms.untracked.double(0.89),
+            eta_cut = cms.untracked.double(2.4),
+            pt_cut = cms.untracked.double(30)
+            )
+        ),
+
+    bTagsTight = cms.PSet(
+        type = cms.string('btags_analyzer'),
+        prefix = cms.string('btags_CSVv2_tight'),
+        enable = cms.bool(True),
+        parameters = cms.PSet(
+            discr_name = cms.untracked.string('pfCombinedInclusiveSecondaryVertexV2BJetTags'),
+            discr_cut = cms.untracked.double(0.97),
+            eta_cut = cms.untracked.double(2.4),
+            pt_cut = cms.untracked.double(30)
+            )
+        ),
+
+    tt = cms.PSet(
+        type = cms.string('tt_analyzer'),
+        prefix = cms.string('tt_'),
+        enable = cms.bool(True),
+        parameters = cms.PSet(
+            electronIsoCut = cms.untracked.double(11),
```

Comment:
Not sure if it's intended to be super loose or it's just a mistake (isolation is stored relative)